product/stream_processor/handlers/process_stream.py

```diff
@@ -6,17 +7,22 @@
 
 from product.models.products.product import ProductChangeNotification
 from product.stream_processor.domain_logic.product_notification import notify_product_updates
+from product.stream_processor.integrations.events.base import BaseEventHandler
 from product.stream_processor.integrations.events.event_handler import EventHandler
 from product.stream_processor.integrations.events.models.output import EventReceipt
 
 logger = Logger()
 
+# NOTE: these will move to environment variables. Event source format could even use a pydantic validation!
+EVENT_BUS = os.environ.get('EVENT_BUS', '')
+EVENT_SOURCE = 'myorg.product.product_notification'
+
 
 @logger.inject_lambda_context(log_event=True)
```

Comment:
I usually tend NOT to debug the entire event for couple of reasons:
1. here its a batch and might be quite large and might get split into multiple logs, i'd rather log the length of the batch and then each item by itself
2. the items might contain PII (i know you have a utility to sort that too coming :))